petition_service/internal/config/config.go

```diff
@@ -23,7 +23,7 @@ type Postgres struct {
 var Cfg Config
 
 func LoadConfig() {
-	data, err := os.ReadFile("petition_service/config.yml")
+	data, err := os.ReadFile("./config.yml")
```

Comment:
not sure what was the intention, dar aici la fel nu trebuie de schimbat nimic; LoadConfig deja e ajustat pentru petitii